website/content/en/preview/faq.md

```diff
@@ -183,7 +183,8 @@ Use your existing upgrade mechanisms to upgrade your core add-ons in Kubernetes
 
 Karpenter requires proper permissions in the `KarpenterNode IAM Role` and the `KarpenterController IAM Role`.
 To upgrade Karpenter to version `$VERSION`, make sure that the `KarpenterNode IAM Role` and the `KarpenterController IAM Role` have the right permission described in `https://karpenter.sh/$VERSION/getting-started/getting-started-with-eksctl/cloudformation.yaml`.
-Next, locate `KarpenterController IAM Role` ARN (i.e., ARN of the resource created in [Create the KarpenterController IAM Role](../getting-started/getting-started-with-eksctl/#create-the-karpentercontroller-iam-role)) and the cluster endpoint, and pass them to the helm upgrade command
+Next, locate `KarpenterController IAM Role` ARN (i.e., ARN of the resource created in [Create the KarpenterController IAM Role](../getting-started/getting-started-with-eksctl/#create-the-karpentercontroller-iam-role)) and pass them to the helm upgrade command. If you are not using EKS you also have to pass the cluster endpoint via `--set settings.aws.clusterEndpoint=${CLUSTER_ENDPOINT}`.
```

Comment:
We're intending these docs to be EKS-specific, so I'm not sure that the additional line is neccesary.